setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+VERSION = '1.0'
+
+from io import open
+from os import path
+from setuptools import setup
+
+here = path.abspath(path.dirname(__file__))
+
+
+def main():
+
+    packages = ['svalinn_tools', 'img']
```

Comment:
What is this `'img'` package?

Reply:
This is where the logo needed by `data_loading.py` is kept! And also an example image for the README.